ios/RNApplePay.m

```diff
@@ -138,6 +169,11 @@ - (void)paymentAuthorizationViewControllerDidFinish:(nonnull PKPaymentAuthorizat
                 self.completeResolve(nil);
                 self.completeResolve = NULL;
             }
+
+            if (self.requestPaymentResolve != NULL) {
+                self.requestPaymentResolve(nil);
+                self.requestPaymentResolve = NULL;
+            }
```

Comment:
I think it should be add in another PR or removed here. Also you should rename this PR because i see "billingContact" changes